### DISCUSSION TITLE
打包整个monorepo项目的时候，如何生成所有子包的source map

### DISCUSSION THREAD

**Discussion by whale2002** [web-infra-dev/modern.js#5511] [Q&A] [answered]
想调试一下modernjs的源码，但是发现默认build不生成source map文件，逐个修改子包的buildConfig又比较繁琐，请问如何简单的生成所有子包的source map文件呢

**Comment by xc2** [comment, marked as answer]
https://github.com/web-infra-dev/modern.js/blob/main/scripts/build/src/index.js
改这个文件
